app.rb

```diff
@@ -0,0 +1,141 @@
+require './person'
+require './student'
+require './teacher'
+require './rental'
+require './book'
+require './classroom'
+require './nameable'
+require './decorator'
+require './capitalizedecorator'
+require './trimmerdecorator'
+
+class App
+  attr_accessor :people, :book, :rental
+
+  def initialize()
+    @people = []
+    @books = []
+    @rentals = []
+    @person = nil
```

Comment:
- [ ] You are not required to have `@person = nil` because you already have `@people = []`